ext/import-font.cpp

```diff
@@ -215,4 +222,48 @@ bool getKerning(double &output, FontHandle *font, unicode_t unicode1, unicode_t
     return getKerning(output, font, GlyphIndex(FT_Get_Char_Index(font->face, unicode1)), GlyphIndex(FT_Get_Char_Index(font->face, unicode2)));
 }
 
+bool setVariationAxis(FontHandle *font, FreetypeHandle *library, const char *name, double coordinate) {
+    bool success = false;
+    if (font->face->face_flags & FT_FACE_FLAG_MULTIPLE_MASTERS) {
+        FT_MM_Var *amaster;
+        FT_Get_MM_Var(font->face, &amaster);
+
+        std::vector<FT_Fixed> coords;
+        coords.resize(amaster->num_axis);
+
+        FT_Get_Var_Design_Coordinates(font->face, coords.size(), coords.data());
+        for (FT_UInt i = 0; i < amaster->num_axis; i++) {
+            int strdiff = strcmp(name,amaster->axis[i].name);
+            if (strdiff == 0) {
+                coords[i] = DOUBLE_TO_F16DOT16(coordinate);
+                success = true;
+                break;
+            }
+        }
+        FT_Set_Var_Design_Coordinates(font->face, coords.size(), coords.data());
+        FT_Done_MM_Var(library->library, amaster);
+    }
+    return success;
+}
+
+bool getVariationAxes(std::vector<FontVariationAxis> &axes, FontHandle *font, FreetypeHandle *library) {
+    if (font->face->face_flags & FT_FACE_FLAG_MULTIPLE_MASTERS) {
+        FT_MM_Var *amaster;
+        FT_Get_MM_Var(font->face, &amaster);
+
+        for (FT_UInt i = 0; i < amaster->num_axis; i++) {
+            FontVariationAxis axis{
+                amaster->axis[i].name,
```

Comment:
Is it guaranteed that the pointer to name string remains valid after `FT_Done_MM_Var`?